README.md

```diff
@@ -2,6 +2,7 @@
 
 OpenTomb — an open-source Tomb Raider 1-5 engine remake
 -------------------------------------------------------
+- Windows: [Release](https://github.com/TokyoSU/OpenTomb/releases) & [Sources](https://github.com/TokyoSU/OpenTomb_Windows)
```

Comment:
destructive README.md changes
the project based on MinGW and GCC building

Reply:
i will revert to original and dont touch it again, sorry for it.

Reply:
#FIXED